js/util.js

```diff
@@ -402,6 +402,82 @@
     }
 
 
+    /*
+     * Motion mini-module.
+     *
+     * Call startTrackingMotion(). The direction property will be
+     * updated.
+     *
+     * Events:
+     *
+     */
+    var motionModule  = (function () {
+        var direction = "",
+            motionModule;
+
+        motionModule = {
+            /**
+             * Starts fetching the motion, periodically, and sets the
+             * direction property of this module. 
+             */
+            startTrackingMotion: function () {
+                if (watchID !== null)
```

Comment:
oops, doesn't refer to anything, I don't actually see a use for this line here at all, so I'll delete it.